src/FabCatalog.js

```diff
@@ -10,20 +10,21 @@ class Fab extends Component {
   }
 
   render() {
-    const {mini} = this.props;
-    const classes = `mdc-fab material-icons ${mini ? 'mdc-fab--mini' : ''}`;
+    const classes = `mdc-fab
+      ${this.props.mini ? 'mdc-fab--mini ' : ''}
+      ${this.props.extended ? 'mdc-fab--extended ' : ''}
+      ${this.props.classes ? this.props.classes : ''}`;
+    const ariaLabel = this.props.ariaLabel || this.props.textLabel;
```

Comment:
Why are you creating a textLabel? Why not just use ariaLabel again?

Reply:
Didn't want to keep same param names for aria label and text label.

For example, Standard FAB uses only aria label where as Extended FAB with icon sends text label which is used in aria and text label.